### PR TITLE
Bump markupsafe

### PR DESCRIPTION
**Pull request by jkleung11**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,38 +11,39 @@
...
+    'pytest-runner==6.0.0',
+    'Jinja2==3.0.3',
+    'MarkupSafe==2.0.1'
 ]
 
 setup(
     author="Scott Rothbarth",
     author_email='srserves85@gmail.com',
-    python_requires='>3.5, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
+    python_requires='>3.6, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, !=3.6.*',
```
Comment (ilyaberd) [inline setup.py]: This seems wrong" >3.6 AND != lower versions? Should just be >3.6?

Comment (jkleung11) [reply setup.py]: yeah i was thinking that to, going to experiment with just using >

Comment (ilyaberd) [reply setup.py]: Got it. It should be: python_requires='>3.6, !=3.6.*'

File: requirements_dev.txt [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+pip==22.0.4,
+bump2version==1.0.1,
+wheel==0.37.1,
+watchdog==2.1.6,
+flake8==4.0.1,
+tox==3.24.5,
+coverage==6.3.2,
+Sphinx==4.4.0,
+twine==3.8.0,
+Click==8.0.4,
```
Comment (ethan-schwab) [inline requirements_dev.txt:10]: i might have missed this last week, how did you figure out which versions to change all these to?

Comment (jkleung11) [reply requirements_dev.txt:10]: i went full send and updated all of them

File: setup.py (hunk 1 above)
Comment (ilyaberd) [reply setup.py]: Yes, just >3.6

**Pull request merged**